tests/SignalSlot/RemoveTranslationSlotTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\PlatformHttpCacheBundle\Tests\SignalSlot;
+
+use EzSystems\PlatformHttpCacheBundle\SignalSlot\RemoveTranslationSlot;
+use eZ\Publish\Core\SignalSlot\Signal\ContentService\RemoveTranslationSignal;
+use eZ\Publish\SPI\Persistence\Content\Location;
+use eZ\Publish\SPI\Persistence\Content\Location\Handler as LocationHandler;
+
+/**
+ * Test RemoveTranslationSlot for HttpCache.
+ */
+class RemoveTranslationSlotTest extends AbstractContentSlotTest
+{
+    protected $locationId = 43;
+    protected $parentLocationId = 45;
+
+    /**
+     * @var \eZ\Publish\SPI\Persistence\Content\Location\Handler|\PHPUnit_Framework_MockObject_MockObject
+     */
+    private $locationHandlerMock;
+
+    public function setUp()
+    {
+        $this->locationHandlerMock = $this->getMock(LocationHandler::class);
+        parent::setUp();
+    }
+
+    /**
+     * @dataProvider getReceivedSignals
+     * @param \eZ\Publish\Core\SignalSlot\Signal $signal
+     */
+    public function testReceivePurgesCacheForTags($signal)
+    {
+        $this->locationHandlerMock
+            ->expects($this->once())
+            ->method('loadLocationsByContent')
+            ->with($this->contentId)
+            ->willReturn(
+                [
+                    new Location(
+                        [
+                            'id' => $this->locationId,
+                            'parentId' => $this->parentLocationId,
+                        ]
+                    ),
+                ]
+            );
+
+        parent::testReceivePurgesCacheForTags($signal);
+    }
+
+    public function createSignal()
+    {
+        return new RemoveTranslationSignal(
```

Comment:
Oh, shouldn't we skip the test if class is not present @alongosz? otherwise it will fail on prev branches right?